templates/user_create.html.j2

```diff
@@ -9,6 +9,9 @@
                     <div class="box">
                         <article class="message is-danger" x-show="error != ''">
                             <div class="message-body" x-text="error"></div>
+                        </article>
+                         <article class="message is-success" x-show="showRedirectToPools">
```

Comment:
Place this outside of above div. Show only this message.

Reply:
done